litellm/proxy/proxy_cli.py

```diff
@@ -19,8 +19,6 @@
 
 
 def append_query_params(url, params):
-    print(f"url: {url}")
-    print(f"params: {params}")
```

Comment:
instead of removing can we use `verbos_proxy_logger.debug` for this

Reply:
Sure, let me make the change